common-healing.lic

```diff
@@ -249,13 +249,34 @@ module DRCH
   end
 
   def bind_wound(body_part, person = 'my')
+    # Either you tended it or it already has bandages on. Good job!
+    tend_success = [
+      'You work carefully at tending',
+      'That area has already been tended to',
+      'That area is not bleeding'
+    ]
+    # Yeouch, you couldn't tend the wound. Might have made it worse!
+    tend_failure = [
+      'You fumble',
+      'too injured for you to do that'
+    ]
+    # You dislodged ammo or a parasite, discard it.
+    tend_dislodge = [
+      'You \w+ remove'
+    ]
     snap = [DRC.left_hand, DRC.right_hand]
-    if /You \w+ remove/ =~ DRC.bput("tend #{person} #{body_part}", 'You work carefully at tending', 'That area has already been tended to', 'That area is not bleeding', 'You fumble', 'too injured for you to do that', 'You \w+ remove')
```

Comment:
Moved the match strings into arrays to avoid duplicating them in the case/when statements